WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -70,4 +72,23 @@ + (NSArray *)routes
   return FBResponseWithOK();
 }
 
++ (id<FBResponsePayload>)handleDismissKeyboardCommand:(FBRouteRequest *)request
+{
+  [request.session.application dismissKeyboard];
```

Comment:
sure, thanks for the suggestion. 
And yes, we need this method in Appium, since we had the _hideKeyboard_ function in our API from the beginning and right now it became hacky and slow. I'd like to just have one native call there instead.